vita3k/modules/SceAppMgr/SceAppMgr.cpp

```diff
@@ -445,9 +445,24 @@ EXPORT(int, _sceAppMgrLoopBackMount) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, _sceAppMgrMmsMount) {
-    TRACY_FUNC(_sceAppMgrMmsMount);
-    return UNIMPLEMENTED();
+EXPORT(SceInt32, _sceAppMgrMmsMount, SceInt32 id, char *mount_point) {
+    TRACY_FUNC(_sceAppMgrMmsMount, id, mount_point);
+    switch (id) {
+    case 0x190:
+        strcpy(mount_point, "ux0:mms/photo/");
+        break;
+    case 0x191:
+        strcpy(mount_point, "ux0:mms/music");
```

Comment:
Why is there a / at the end of photo and not music?

Reply:
Is old, but problably because path already have it and another not 🤔 i remember one thinks like that

Reply:
removed it, used by photos/viodeo app and no have view problem